### ISSUE TITLE
No example again

### ISSUE THREAD

**Issue by web-bert** [circleci/circleci-docs#1103]
This issue is about <https://circleci.com/docs/2.0/project-walkthrough/> (source file: <https://github.com/circleci/circleci-docs/blob/master/jekyll/_cci2/project-walkthrough.md>)

You reference
```yaml
- image: selenium/standalone-chrome:3.1.0
```

Yet no where to do address a) why you have this and b) how the hell to use it

I'm finding it increasingly frustrating trying to work out how to run some simple selenium tests in Circle CI

**Comment by michelle-luna** [comment]
Hi,

Thanks for this feedback on the beta docs, here is a link to a thread on discuss requesting something similar, in case it helps you: https://discuss.circleci.com/t/starting-xvfb-for-selenium/12763/5 

I will get a doc added to cover this in 2.0 (I don't think the project walkthrough is the right place to add this info, but let me know what you think).

-Michelle

**Comment by web-bert** [comment]
Another day lost to this. What I can't seem to get working (and what you don't explain anywhere) is a) how do you start a Node.js app running in the background and then b) what url do you use to enable the remote selenium to access the app?

I have tried all manner of things to get the app running and I can't for the life of me get any meaningful logs output anywhere to work out what the hell is going on.

What I also don't understand is what docker vm my code is running in? You used to have: 
```yaml
machine:
  node:
    version: 6.9.1
```
and that seems to be replaced by:
```yaml
version: 2
jobs:
  build:
    docker:
      - image: circleci/node:6.10
```
So do you use the first image in the docker list to use as the vm to checkout the code into? Do you look for a circleci image and run the code in that? It really isn't explained anywhere.

So for now I give up and we won't be running any automated tests as it doesn't appear to work - and there isn't a clear guide to show you how to make it work.

**Comment by michelle-luna** [comment]
Hi,

Thanks for these questions, it will help me improve the docs to know where you are stuck, much appreciated. For a) and b) there is some useful info and options in this thread that you can try:
https://discuss.circleci.com/t/run-http-server-in-background-during-tests/3177/2

Yes, the first image is the vm (we call it container) used for checkout. 

-M

**Comment by web-bert** [comment]
That doesn't provide any useful info as the main question is still not answered.

From what I have figured out with lots of trial and error: all the containers that are started are all mapped to "localhost". So I have been able to start my app on port 8080 and when connecting to the standalone-chrome remote server I just issue a command to connect to "http://localhost:8080/" to access my app (even though it is one container communicating to another).

Although now I have issues with one container talking to my postgres container. I assume that it should us the hostname "localhost" on the default postgres port. But again, no information on how the network is setup for the docker containers to communicate.

It would be really handy to be able to see the logs from the docker containers, but when the "circleci build" command finishes it deletes the containers so I can't access the logs :-/
